### DISCUSSION TITLE
HMT-2250-6T

### DISCUSSION THREAD

**Discussion by letmoc** [tbnobody/OpenDTU#872] [General]
Hello, is it possible to output the voltages of the 3 phases separately via the openDTU? 
Currently, only around 400 volts are output.

**Comment by h3tz** [comment]
Sure. Via Python: https://github.com/tbnobody/OpenDTU/blob/master/docs/Web-API.md

**Comment by thoutenbos** [comment]
I have the same inverter and the same question…
Via the webview and the API only 1 AC phase is displayed.
How can the info for the other phases be seen?

**Comment by thoutenbos** [comment]
In the Hoymiles DTU it is possible to see all three phases, and it shows the normal P+N voltages instead of the P+P voltage (~400V) which is shown by OpenDTU.

How can we correctly display the 3P grid status in OpenDTU?

![IMG_0035](https://github.com/tbnobody/OpenDTU/assets/5472093/040beb6a-b68c-4987-a555-c82e36ae4317)

**Comment by MiG-41** [comment]
Still not possible ?
